apps/settings/js/panels/root/panel.js

```diff
@@ -158,3 +133,28 @@ define(function(require) {
     });
   };
 });
+
+/**
+ * Update the sim and NFC related items based on mozMobileConnections and Device.
+ */
+
+var updateSimItemsAndNfc = function rph_refrehsSimItems() {
```

Comment:
This Panel already having functions related to connectivity settings earlier, this js file loaded on main page in 2.2r but now connectivity is moved to second page and this file too .So that's why we place this function in this file.